ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -540,7 +540,7 @@ data Apps m addr bCS bBF bTX bKA bPS a b = Apps
 --
 -- They don't depend on the instantiation of the protocol parameters (which
 -- block type is used, etc.), hence the use of 'RankNTypes'.
-data ByteLimits bCS bBF bTX bKA = ByteLimits
+data ByteLimits addr bCS bBF bTX bKA bPS = ByteLimits
```

Comment:
I think it would be more in line to remove this and rather quantify in the type of `blPeerSharing`, also see the comment

```haskell
-- They don't depend on the instantiation of the protocol parameters (which
-- block type is used, etc.), hence the use of 'RankNTypes'.
```
just above.

```suggestion
data ByteLimits bCS bBF bTX bKA bPS = ByteLimits
```